examples/course-minimal/qwiklabs.yaml

```diff
@@ -42,13 +42,17 @@ badge: gcp-intro-course-badge.png
 
 estimated_duration_days: 1
 
-steps:
-  - activity_options:
-    - type: lab
-      content: intro-to-gcp
-  - activity_options:
-    - type: lab
-      content: intro-to-bigquery
-  - activity_options:
-    - type: quiz
-      content: gcp-quiz
+modules:
+  - title:
+      en: Module 1
```

Comment:
Please nest this under `locales` to match other localized bundle content.

Reply:
Good catch. Will update shortly.